src/github.com/couchbase/sync_gateway/rest/admin_api_test.go

```diff
@@ -908,3 +908,71 @@ func (rt *restTester) createSession(t *testing.T, username string) string {
 
 	return sessionId
 }
+
+
+func TestPurgeWithBadJsonPayload(t *testing.T) {
+	var rt restTester
+	response := rt.sendAdminRequest("POST", "/db/_purge", "foo")
+	assertStatus(t, response, 400)
+}
+
+func TestPurgeWithNonArrayRevisionList(t *testing.T) {
+	var rt restTester
+	response := rt.sendAdminRequest("POST", "/db/_purge", `{"foo":"list"}`)
+	assertStatus(t, response, 200)
+}
+
+func TestPurgeWithEmptyRevisionList(t *testing.T) {
+	var rt restTester
+	response := rt.sendAdminRequest("POST", "/db/_purge", `{"foo":[]}`)
```

Comment:
Is there more we can be validating for some of these tests (e.g. the number of docs purged, etc)?